static/app/components/events/interfaces/crashContent/exception/setupSourceMapsAlert.spec.tsx

```diff
@@ -1,61 +1,105 @@
+import cloneDeep from 'lodash/cloneDeep';
+
 import {initializeOrg} from 'sentry-test/initializeOrg';
 import {render, screen} from 'sentry-test/reactTestingLibrary';
 
+import {Event} from 'sentry/types';
+
 import {SetupSourceMapsAlert} from './setupSourceMapsAlert';
 
+const alertText = 'Sentry can un-minify your code to show you more readable stack traces';
+
+function modifyEventFrames(event: Event, modify: any): Event {
+  const modifiedEvent = cloneDeep(event);
+  modifiedEvent.entries[0].data.values[0].stacktrace.frames =
```

Comment:
do we know exactly that we will have one element here ^ ?

Reply:
yeah, it's a test fixture 👍